clang-tools-extra/clangd/SemanticSelection.cpp

```diff
@@ -181,16 +181,16 @@ llvm::Expected<std::vector<FoldingRange>> getFoldingRanges(ParsedAST &AST) {
 // Related issue: https://github.com/clangd/clangd/issues/310
 llvm::Expected<std::vector<FoldingRange>>
 getFoldingRanges(const std::string &Code, bool LineFoldingOnly) {
```

Comment:
Should we revert 70914aa631561aa9a29681bfe5159b4ea6952060 instead?

Reply:
I think we should keep it in this patch to avoid changing the current behavior of clangd.